src/main/java/seedu/task/storage/JsonAdaptedTask.java

```diff
@@ -77,6 +80,7 @@ public JsonAdaptedTask(@JsonProperty("name") String name,
     public JsonAdaptedTask(Task source) {
         name = source.getName().fullName;
         description = source.getDescription().value;
+        hasDescription = String.valueOf(source.getDescription().getHasDescription());
```

Comment:
Would it be better to abstract out this method as per law of demeter?

Reply:
Updated as suggested.